src/widget/whighlightinggroup.h

```diff
@@ -0,0 +1,59 @@
+#ifndef WHIGHLIGHTINGGROUP_H
+#define WHIGHLIGHTINGGROUP_H
+
+#include "widget/wwidgetgroup.h"
+
+class SkinContext;
+
+// The WHighlightingGroup can be used to highlight a group of widgets by
+// changing the background or any other css style option.
+// Example skin:
+// <HighlightingGroup>
+//  <Id>1</Id>
+//  <BackPathHighlighted>style/style_bg_effect1_high</BackPathHighlighted>
+//  <Connection>
+//   <ConfigKey>[EffectRack1_EffectUnit1],single_effect_focus</ConfigKey>
+//   <BindProperty>highlightedId</BindProperty>
+//  </Connection>
+// </HighlightingGroup>
+
+class WHighlightingGroup : public WWidgetGroup {
+    Q_OBJECT
+  public:
+    explicit WHighlightingGroup(QWidget* pParent = nullptr);
+
+    // The highlight property is used to restyle the widget with CSS.
+    // The declaration #MyGroup[highlight="1"] { } will define the style
+    // for the highlighted state. Note: The background property does not
+    // support color schemes for images, a workaround is to set the background
+    // image via <BackPath> and <BackPathHighlighted> from the skin.
+    Q_PROPERTY(int highlight READ getHighlight WRITE setHighlight NOTIFY highlightChanged)
+
+    // The highlightedId property is used to select the highlighted widget
+    // group by its Id, which can be set from skin by <Id>4</Id>
+    Q_PROPERTY(int highlightedId READ getHighlight WRITE setHighlightedId NOTIFY highlightChanged)
```

Comment:
Since is optional, you can use the ID, as shown in this PR or not like you describe. 

The use-case for the ID value works like that: 

Creat n HighligtingGroups and name them EffectBackground. Set the ID of the first HighligtingGroup to 1 and up to the n'th HighligtingGroup to n. Bind the highlightedId property of each to the same  CO, wich can have the values 0 ... n 

Now you can style all these groups like that: 

```
#EffectBackground {
 /*default style*/
}
#EffectBackground[highlight="1"] {
 /*higlighted style*/
}
```

Without that ID feature, each of the EffectBackgrounds need an individual name and an individual style. The color schema changing background will similar hard to use since you have to define a background for each of the N states. 

I have made this feature separate, because it is more attracting for the user and a bit resource saving RAM/CPU. Remember that the majority of groups are happy with the default QT states. 
But If you think it is a benefit to merge HighligtingGroups and WidgetGroups I can do it. 
Should I?